src/app/learn/get-started/prereqs/ssl-certificate/index.html

```diff
@@ -213,18 +234,18 @@
   </ol>
 
   <stache-page-anchor>
-    Installation for Linux
+    Manual installation for Linux
   </stache-page-anchor>
 
   <ol>
     <li>
-      <p>
-        Download the <a href="https://raw.githubusercontent.com/blackbaud/skyux-builder/master/ssl/skyux-ca.crt">skyux-ca.crt</a>. To download, right-click the link and save it to your computer.
-      </p>
+      <sky-code-block>
```

Comment:
Same note above about the clunkiness of the code block in the step. It seems like it should just be a step that says "Run `skyux certs generate`."

Reply:
Agreed.